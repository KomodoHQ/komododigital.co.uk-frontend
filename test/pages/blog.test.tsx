import React from 'react';
//import Blog from '../../src/pages/blog';
import renderer from 'react-test-renderer';
import GatsbyData from '../../gatsby-config';

test('Blog page is rendered', () => {
  const data = {
    site: GatsbyData,
    allWordpressPost: {
      edges: [
        {
          node: {
            featured_media: {
              source_url: 'https://www.komododigital.co.uk/static/media/logo.50a4b652.svg',
            },
            content: 'TEST CONTENT',
            title: 'TEST TITLE',
          },
        },
      ],
    },
  };
  // const component = renderer.create(<Blog data={data} />);
  // let tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
});
