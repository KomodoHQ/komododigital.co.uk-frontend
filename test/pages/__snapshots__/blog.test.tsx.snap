// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog template is rendered 1`] = `
<div>
  <img />
  <h1 />
  <div
    dangerouslySetInnerHTML={
      Object {
        "__html": "<p>Test</p>",
      }
    }
  />
</div>
`;
